styleguide/assets/additional.css

```diff
@@ -1,9 +1,8 @@
-@import 'server.css';
+@import url('https://nextcloud.github.io/server/apps/theming/css/default.css');
+@import url('https://nextcloud.github.io/server/core/css/server.css');
```

Comment:
The issue is track and versioning css changes.

Reply:
We keep updating it manually to master anyway :shrug:

Reply:
Only adds the dependency on keeping the sass compiled css files in the repo, but think that is fine.

Reply:
We already have that depepdency. There is also tests on server to make sure this file is up to date.

Reply:
Sure, just wanted to have that mentioned if we decide to move that out and make it part of the release process rather than having build assets in the repo. :) I'm definitely all in for doing that now so we have more up to date css on the vue components. 👍

Reply:
> We already have that depepdency. There is also tests on server to make sure this file is up to date.

We have a task to remove the compiled assets from server (hopefully soon), so this might break at some point.

Reply:
No, this one needs to be in for error pages.
But luckily it should not change that frequently as css server variables should be somewhat stable